### PR TITLE
WatchWidget: Add new/delete/clear toolbar items

### PR DESCRIPTION
**Pull request by sepalani**
This PR adds to the Watch widget toolbar items to create, delete and clear watches.

Ready to reviewed & merged.

### CODE REVIEW

File: Source/Core/DolphinQt/Debugger/WatchWidget.cpp [hunk 1]
```diff
@@ -111,6 +118,10 @@ void WatchWidget::ConnectWidgets()
 
 void WatchWidget::UpdateIcons()
 {
+  // TODO: Create a "debugger_add_watch" icon
```
Comment (iwubcode) [inline Source/Core/DolphinQt/Debugger/WatchWidget.cpp:121]: Who would create this?  How do we keep track of this other than a code "TODO"?  Maybe an issue in the issue tracker?  I have some icons needed too..

Comment (JosJuice) [reply Source/Core/DolphinQt/Debugger/WatchWidget.cpp:121]: @MayImilae would be the person to talk to.

Comment (iwubcode) [reply Source/Core/DolphinQt/Debugger/WatchWidget.cpp:121]: @JosJuice - is there a way we can track this or must we block this PR until MayImilae has time to create said icons?

Comment (JosJuice) [reply Source/Core/DolphinQt/Debugger/WatchWidget.cpp:121]: I don't think we really have a standard way to track icon requests... But I suppose the issue tracker could make sense.

**Comment by JMC47** [comment]
May said she doesn't have time to work on an icon right now, so I think it is fine to merge it without one for now.

**Pull request merged**